### PR TITLE
Update Next Titles for Algolia indexing

### PR DESCRIPTION
**Pull request by jay-khatri**
## Summary
Update Nextjs Titles for Algolia indexing

## How did you test this change?
click

## Are there any deployment considerations?
n/a

## Does this work require review from our design team?
n/a

### CODE REVIEW

File: docs-content/getting-started/3_client-sdk/2_nextjs.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Next.js
+title: Next.js Quick-start
```
Comment (deltaepsilon) [inline docs-content/getting-started/3_client-sdk/2_nextjs.md:2]:
```suggestion
title: Next.js Quick Start
```

File: docs-content/getting-started/3_client-sdk/2_nextjs.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 slug: nextjs
-heading: Using highlight.io with Next.JS
+heading: Next.js Quick-start
```
Comment (deltaepsilon) [inline docs-content/getting-started/3_client-sdk/2_nextjs.md:4]:
```suggestion
heading: Next.js Quick Start
```

File: go.work.sum [hunk 3]
```diff
@@ -785,6 +785,7 @@ github.com/googleapis/gnostic v0.5.5 h1:9fHAtK0uDfpveeqqo1hkEZJcFvYXAiCN3UutL8F9
 github.com/gorilla/handlers v1.4.2 h1:0QniY0USkHQ1RGCLfKxeNHK9bkDHGRYGNDFBCS+YARg=
 github.com/gorilla/mux v1.8.0 h1:i40aqfkR1h2SlN9hojwV5ZA91wcXFOvkdNIeFDP5koI=
 github.com/goware/emailproviders v0.0.0-20200728124719-451ef785cf29 h1:/jHkBu2jbrW6+pxmAMt4tR8W6l/Vk02mnGL/ePrg2+I=
+github.com/goware/emailproviders v0.0.0-20200728124719-451ef785cf29/go.mod h1:2FoXdMtWJEKk3zBo2S5eaduD5Bh/5PIYwSlV66E1XMs=
```
Comment (deltaepsilon) [inline go.work.sum:788]: `git reset` this file and run `git submodule update --recursive` to reset the `opentelemetry-sdk-workers` submodule.

**Review by deltaepsilon — CHANGES_REQUESTED** [review]

File: docs-content/getting-started/3_client-sdk/2_nextjs.md (hunk 1 above)
Comment (deltaepsilon) [reply docs-content/getting-started/3_client-sdk/2_nextjs.md:2]: These `Quick-start` to `Quick Start` edits are optional, but I wanted to make sure that the other comment was addressed, hence the "requested changes".

**Review by Vadman97 — APPROVED** [review]

**Pull request merged**